README.md

```diff
@@ -6,33 +6,42 @@
 
 To run the chatbot, then it is necessary to have `Node` installed on your system.
 
-The project has only been tested on recent versions of Node, and hence it is recommened to use Node `>= 8.0`.
+The project has been tested on multiple versions of Node, and it is recommened to use Node versions between `8.0` and `10.13`.
 
 ### Typescript
 
 Furthermore typescript is required to compile the project to Node compatible JavaScript code.
 
 To install TypeScript run the command `npm install -g typescript` or `yarn global add typescript`
 
+### Yarn
+
+In case you want to use yarn make sure you have it installed by running the command `npm install --global yarn`
```

Comment:
Is Yarn not required to run the project?

Reply:
You can just use npm to call the scripts instead of yarn.

Reply:
Changed wording